core/web/keys_controller.go

```diff
@@ -1,6 +1,7 @@
 package web
 
 import (
+	"fmt"
```

Comment:
is there a reason we can't use `errors.Errorf`?

Reply:
If so, is there any benefit to doing so?

Reply:
:shrug: Would it be an improvement in this case? The panic will already have the same stack trace.

Reply:
panic doesn't even need an error actually - it's quite common to pass just a formatted string.